tests/testWalkCycle.cpp

```diff
@@ -52,19 +52,25 @@ TEST(WalkCycle, inverse_kinematics) {
   phase1.addContactPoint("lower3", contact_in_com);  // RH
   auto walk_cycle = WalkCycle({phase0, phase1});
 
-  // Set goal points to reasonable values
-  Point3 goal_LH(0, 0.15, 0);     // LH
-  Point3 goal_LF(0.6, 0.15, 0);   // LF
-  Point3 goal_RF(0.6, -0.15, 0);  // RF
-  Point3 goal_RH(0, -0.15, 0);    // RH
+  // Expected contact goal points.
+  Point3 goal_LH(-0.371306, 0.1575, 0);   // LH
+  Point3 goal_LF(0.278694, 0.1575, 0);    // LF
+  Point3 goal_RF(0.278694, -0.1575, 0);   // RF
+  Point3 goal_RH(-0.371306, -0.1575, 0);  // RH
+
+  // Check initalization of contact goals.
+  auto cp_goals = walk_cycle.initContactPointGoal(robot, -0.191839);
```

Comment:
Random value of `-0.191839`?

Reply:
No, height of feet in rest config from SDF